src/client/eppo-client.ts

```diff
@@ -141,21 +145,27 @@ export default class EppoClient implements IEppoClient {
   private isGracefulFailureMode = true;
   private assignmentCache: AssignmentCache<Cacheable> | undefined;
   private configurationStore: IConfigurationStore;
-  private configurationRequestConfig: ExperimentConfigurationRequestParameters | undefined;
+  private configurationRequestParameters: ExperimentConfigurationRequestParameters | undefined;
   private requestPoller: IPoller | undefined;
 
   constructor(
     configurationStore: IConfigurationStore,
-    configurationRequestConfig?: ExperimentConfigurationRequestParameters,
+    configurationRequestParameters?: ExperimentConfigurationRequestParameters,
   ) {
     this.configurationStore = configurationStore;
-    this.configurationRequestConfig = configurationRequestConfig;
+    this.configurationRequestParameters = configurationRequestParameters;
+  }
+
+  public setConfigurationRequestParameters(
+    configurationRequestParameters: ExperimentConfigurationRequestParameters,
+  ) {
+    this.configurationRequestParameters = configurationRequestParameters;
   }
 
   public async fetchFlagConfigurations() {
-    if (!this.configurationRequestConfig) {
+    if (!this.configurationRequestParameters) {
       throw new Error(
-        'Eppo SDK unable to fetch flag configurations without a request configuration',
+        'Eppo SDK unable to fetch flag configurations without a configuration request parameters',
```

Comment:
Parameters is plural so "Eppo SDK unable to fetch flag configurations without ~a~ configuration request parameters"